src/overlays/actors/ovl_Bg_Hidan_Dalm/z_bg_hidan_dalm.c

```diff
@@ -21,17 +29,175 @@ const ActorInit Bg_Hidan_Dalm_InitVars = {
     (ActorFunc)BgHidanDalm_Update,
     (ActorFunc)BgHidanDalm_Draw,
 };
-*/
+
+static ColliderTrisItemInit sTrisItemInit[4] = {
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000040, 0x00, 0x00 }, 0x00, 0x79, 0x00 },
+        { { { 305.0f, 0.0f, -300.0f }, { 305.0f, 600.0f, -300.0f }, { 305.0f, 600.0f, 300.0f } } },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000040, 0x00, 0x00 }, 0x00, 0x79, 0x00 },
+        { { { 305.0f, 0.0f, -300.0f }, { 305.0f, 600.0f, 300.0f }, { 305.0f, 0.0f, 300.0f } } },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000040, 0x00, 0x00 }, 0x00, 0x79, 0x00 },
+        { { { -305.0f, 0.0f, -300.0f }, { -305.0f, 600.0f, 300.0f }, { -305.0f, 600.0f, -300.0f } } },
+    },
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000040, 0x00, 0x00 }, 0x00, 0x79, 0x00 },
+        { { { -305.0f, 0.0f, -300.0f }, { -305.0f, 0.0f, 300.0f }, { -305.0f, 600.0f, 300.0f } } },
+    },
+};
+
+#define sTrisInit D_80886590
+ColliderTrisInit sTrisInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x00, 0x20, COLSHAPE_TRIS },
+    4,
+    sTrisItemInit,
+};
+
+#define sInitChain D_808865A0
+InitChainEntry sInitChain[] = {
```

Comment:
is the define going to be local to this file? it would cause issues if not, just making sure.

generally we just dont rename the var and leave it as the D_ variant until the asm file isnt needed anymore.
But if this solution works its probably fine (@Roman971 thoughts?)